content/en/modules/testing/index.md

```diff
@@ -32,57 +32,29 @@ The prerequisites to take this module are:
+The [slides](https://drive.google.com/file/d/1M3nr4D0-cPCjHjL23vk-BXth2TqHshtj/view?usp=sharing) are adapted from the original QLSC 612 course in 2020 by [Greg Kiar](https://twitter.com/g_kiar)
 
 The video of the presentation is available below:
-<iframe width="560" height="315" src="https://www.youtube.com/embed/VibDC49ZAJE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/TIPIap8rZyE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
 
 
 ## Exercise
 
- * Watch the video until **55:23**.
- * Fork [this repo](https://github.com/FrancoisPgm/intro2testing).
- * On your fork in github, go to the **Actions** tab and create a new workflow.
- * Modify the action template created by github to :
-    * Change the workflow name from "build" to "test"
-    * add the following step to setup python :
-    ```
-          # Set up python to be able to run python scripts
-          - name: Set up Python
-            uses: actions/setup-python@v1
-            with:
-              python-version: 3.7
-    ```
-    * add the following step to install the dependancies :
-    ```
-          # Pip install the required packages
-          - name: Install Dependencies
-            run: pip install -r requirements.txt
-    ```
-    * Run the test script :
-    ```
-          # Run the test script
-          - name: Run test
-            run: |
-              cd code
-              bash test.sh
-    ```
- * commit and push this new file to create and run the action.
- * go back to the action tab on your repo, you should see an action called "test".
- * verify that the action did run the test, the first one should be passed and the second should fail, so you should be able to see something like that :
-
-    <img src="example_action.png" alt="Action example" width="500"/>
+ * Fork [this repo](https://github.com/FrancoisPgm/testing_CI_module) for the hands on part.
+ * Watch the video, and follow along the hands on implementing the unit tests and the github action to run them.
  * Follow up with François Paugam to validate you completed the exercise correctly.
  * :tada: :tada: :tada: you completed this training module! :tada: :tada: :tada:
 
+ **Bonus and addendum** : During the first part of the video I said we would implement unit tests, an integration test and an installation test. I forgot about the integration test during the hands on so I leave it t you as a bonus exercise, which corresponds to writing a test for the main function, since it integrates the other three functions. For the installation test, this one comes for free with the github action, as the action runs in a virtual machine and has to re-install the dependancies each time. 
```

Comment:
```suggestion
 **Bonus and addendum** : During the first part of the video I said we would implement unit tests, an integration test and an installation test. I forgot about the integration test during the hands on so I leave it to you as a bonus exercise, which corresponds to writing a test for the main function, since it integrates the other three functions. For the installation test, this one comes for free with the github action, as the action runs in a virtual machine and has to re-install the dependencies each time. 
```